lib/rototiller/task/params/option.rb

```diff
@@ -40,6 +40,11 @@ def add_argument(*args, &block)
       def to_str
         [@name.to_s, @arguments.to_s].compact.join(' ')
       end
+
+      def stop
+        return true if @arguments.stop?
+        return true unless @name
+      end
```

Comment:
An env_var can thing the task needs to stop, however if a param has a name it no longer stops.  The only time we ask if a collection of env_vars should stop is when it is attached to task.